modules/cluster-logging-elasticsearch-audit.adoc

```diff
@@ -18,67 +18,35 @@ The internal {product-title} Elasticsearch log store does not provide secure sto
 
 To use the Log Forward API to forward audit logs to the internal Elasticsearch instance:
 
-. If the Log Forward API is not enabled:
-
-.. Edit the Cluster Logging Custom Resource (CR) in the `openshift-logging` project:
-+
-----
-$ oc edit ClusterLogging instance
-----
-
-.. Add the `clusterlogging.openshift.io/logforwardingtechpreview` annotation and set to `enabled`:
-+
-[source,yaml]
-----
-apiVersion: "logging.openshift.io/v1"
-kind: "ClusterLogging"
-metadata:
-  annotations:
-    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
-  name: "instance"
-  namespace: "openshift-logging"
-spec:
-
-...
-
-  collection: <2>
-    logs:
-      type: "fluentd"
-      fluentd: {}
-----
-<1> Enables and disables the Log Forwarding API. Set to `enabled` to use log forwarding. 
-<2> The `spec.collection` block must be defined to use Fluentd in the Cluster Logging CR.
-
 . Create a Log Forwarding CR YAML file or edit your existing CR:
 +
 * Create a CR to send all log types to the internal Elasticsearch instance. You can use the following example without making any changes:
 +
 [source,yaml]
 ----
 apiVersion: logging.openshift.io/v1alpha1
```

Comment:
apiVersion: logging.openshift.io/v1

Reply:
@mburke5678  You needn't to defin the outputs sections in ClusterLogForwarder. There is a predefined keyword 'default'
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: instance
  namespace: openshift-logging
spec:
  pipelines:
    - name: test-infra
      inputRefs:
      - infrastructure
      - application
      - audit
      outputRefs:
      - default